### ISSUE TITLE
'float' object is not subscriptable

### ISSUE THREAD

**Issue by keinccgithub** [fmfn/BayesianOptimization#336]
When I run first example in the Basic tour in python, it is wrong as:

```
---> 65 if max_acq is None or -res.fun[0] >= max_acq:
     66     x_max = res.x
     67     max_acq = -res.fun[0]

TypeError: 'float' object is not subscriptable
```

Is the origin code in "bayes_opt" wrong??

**Comment by till-m** [comment]
I think this is fixed in a PR (#303), see also #300 #313 #333.

I guess the question is when a new release will be made that pushes these changes to PI's.

**Comment by till-m** [comment]
For now the best fix for you might be installing from github as mentioned in the readme:
`pip install git+https://github.com/fmfn/BayesianOptimization`
